dist/tools/nacl_key_generator/README.md

```diff
@@ -0,0 +1,13 @@
+# Key-pair generator using [NaCL](https://nacl.cr.yp.to/)
+This program will generate two key-pairs and store them as binary files, using
+tweetNaCl package in RIOT
```

Comment:
you included the source of tweetnacl, so it's not a package in RIOT's sense anymore, is it? What was the idea behind going this route isntead of using `USEPKG=tweetnacl`?